### DISCUSSION TITLE
[Docs] Retrieve the depth image from Pointcloud

### DISCUSSION THREAD

**Discussion by Zumbalamambo** [kornia/kornia#946] [Questions]
I have got the Pointcloud, Camera Intrinsic, and Camera Extrinsic Parameters. Is it possible for Kornia to retrieve the depth image using these data?

**Comment by edgarriba** [comment]
@Zumbalamambo You can achieve it using [`project_points`](https://kornia.readthedocs.io/en/latest/geometry.camera.perspective.html#kornia.geometry.camera.perspective.project_points) + [`remap`](https://kornia.readthedocs.io/en/latest/geometry.transform.html?highlight=remap#kornia.geometry.transform.remap). I'm working on a similar example, I hope after publishing I can provide an example :)

**Comment by Zumbalamambo** [comment]
@edgarriba thank you! could you please share with me a minimal example? :)
